online-dashboard-api/app/Models/User.php

```diff
@@ -76,10 +77,14 @@ public function initiatePasswordReset(): void
 
     public static function createUser($data): object
     {
-        return self::create([
+        $user = self::create([
             'name' => $data['name'],
             'email' => $data['email'],
             'password' => Hash::make($data['password']),
         ]);
+
+        $user->assignRole('user');
```

Comment:
Where the assignRole() functions or did I miss anything!?

Reply:
i didn't get you, but it's HasRoles trait function from spatie pacakage you just need to import the trait in you model then yoiu can use it whenever and whereever you want.

Reply:
Okay! assignRole() this function is driving from the library isn't it! ?

Reply:
yup